projects/core/src/user/facade/user-address.service.ts

```diff
@@ -87,38 +78,30 @@ export class UserAddressService {
    * @param address a user address
    */
   updateUserAddress(addressId: string, address: Address): void {
-    this.authService
-      .getOccUserId()
-      .pipe(take(1))
-      .subscribe(occUserId =>
-        this.store.dispatch(
-          new UserActions.UpdateUserAddress({
-            userId: occUserId,
-            addressId: addressId,
-            address: address,
-          })
-        )
+    this.withUserId(userId =>
+      this.store.dispatch(
+        new UserActions.UpdateUserAddress({
+          userId,
+          addressId,
+          address: address,
```

Comment:
you can leave out the property name here

Reply:
Fixed.